bundles/org.openhab.binding.linky/src/main/java/org/openhab/binding/linky/internal/handler/LinkyHandler.java

```diff
@@ -88,10 +88,18 @@ public void initialize() {
         logger.debug("config username = '{}'", config.username);
 
         try {
-            Request requestLogin = new Request.Builder().url(LOGIN_BASE_URI)
-                    .post(LOGIN_BODY_BUILDER.add("IDToken1", config.username).add("IDToken2", config.password).build())
-                    .build();
-            client.newCall(requestLogin).execute().close();
+            // Request requestLogin = new Request.Builder().url(LOGIN_BASE_URI)
+            // .post(LOGIN_BODY_BUILDER.add("IDToken1", config.username).add("IDToken2", config.password).build())
+            // .build();
+            // client.newCall(requestLogin).execute().close();
+            String requestContent = String.format(LOGIN_BODY_BUILDER,
+                    Base64.getEncoder().encodeToString("realm=particuliers".getBytes(StandardCharsets.UTF_8)),
+                    config.username, config.password);
+            InputStream stream = new ByteArrayInputStream(requestContent.getBytes(StandardCharsets.UTF_8));
+            logger.debug("executeUrl POST {} requestContent {}", LOGIN_BASE_URI, requestContent);
+            HttpUtil.executeUrl("POST", LOGIN_BASE_URI, stream, "application/x-www-form-urlencoded",
```

Comment:
@lolodomo : this does not work. I land with a ``` Exception calling Enedis API : java.io.IOException - java.util.concurrent.ExecutionException: javax.net.ssl.SSLException: Received fatal alert: handshake_failure```

Reply:
Like for the Freebox binding, we probably need to define the public certificate as a trusted certificate.
I don't remember how to retrieve this certificate. Maybe from a WEB browser. 

If you like, switch back to okhttp but keep my code as commented. I will work on this later when my account is opened.

Reply:
I'm going to wait that you have your credentials to test this :)